test/libsolidity/semanticTests/call_forward_bytes.sol

```diff
@@ -0,0 +1,26 @@
+contract receiver {
```

Comment:
I'd avoid adding more tests directly in `semanticTests/`. `semanticTests/fallback/` might be a better place for this one.

Reply:
Moved test case to semanticTests/fallback